### PR TITLE
vdk-jupyter: convert tutorial

### PR DESCRIPTION
**Pull request by duyguHsnHsn**
What: 
Added a tutorial which  shows how to work with the Convert Job Operation on the Jupyter UI. 

This PR is only for getting the tutorial reviewed. 
After the tutorial is reviewed and approved it will be moved to the Examples Wiki page.

### CODE REVIEW

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## Prerequisites
+* Installed VDK, Control Service, Jupyter instance, vdk-jupyterlab-extension, vdk-notebook, vdk-ipython
+* Created and developed a VDK Data Job
+
+## Data Job Conversion Guide
+
+This guide presumes that you have already initialized a job either locally or in the cloud and have subsequently
+downloaded it to your local file system.
+
+### 0. [Optional] Get Familiar With Your Data Job
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:13]: This seems wierdly named steps. 

They are converting an existing job. The presumption is that they are already familiar with that.

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+* Installed VDK, Control Service, Jupyter instance, vdk-jupyterlab-extension, vdk-notebook, vdk-ipython
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:5]: Include links to dictionary , the first time you introduce a term - https://github.com/vmware/versatile-data-kit/wiki/dictionary

Or link to the plugin page (for plugins)

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+Time Commitment: About 5-10 minutes.
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:2]: Add empty line ( as they are rendered on teh same line in markdown now)

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+- Ensure you familiarize yourself with it. It will be located at the top of the notebook.
+- After familiarizing yourself with it, you have the option to delete it.
+
+  ![VDK Conversion Guide](images/guide.png)
+
+- Review the new notebook steps.
+
+  ![VDK Notebook Steps](images/notebook_steps.png)
+
+### 8. [Optional] Confirm The Functionality Of Your Data Job Post-Conversion
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:68]: We should include Restore (from backup) optional step.

**Comment by antoniivanov** [comment]
For the optional steps consider using collapsable blocks 


> **Tip:** to making your guide shorter and easier to skim
> 
> Use collapsable blocks. In Markdown this happens with `<details>` and `<summary>` tags. For example: 
> 
> ```
> <details>
>  <summary>Click to expand</summary>
>  <p>This is the content inside the collapsible block. You can put text, images, or any other HTML elements here.</p>
> </details>
> ```
> <details>
>   <summary>Click to expand</summary>
> This is the content inside the collapsible block. You can put text, images, or any other HTML elements here. 
> </details>

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+This guide presumes that you have already initialized a job either locally or in the cloud and have subsequently
+downloaded it to your local file system.
+
+### 0. [Optional] Get Familiar With Your Data Job
+- **Navigate to the data job directory.**
+- Familiarize yourself with the VDK steps and the file structure of the Data Job.
+
+  ![VDK Initial Job Directory](images/initial_job_dir.png)
+
+### 1. [Optional] Validate Your Data Job's Functionality
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:19]: This really seems more like a Tip like https://github.com/vmware/versatile-data-kit/wiki/Tutorial-Guidelines#2-content

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md (hunk 4 above)
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:68]: This also seems more like a tip https://github.com/vmware/versatile-data-kit/wiki/Tutorial-Guidelines#2-content

File: projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,139 @@
+Learn how to convert a VDK Job with .sql and .py steps into a Job with notebook steps using VDK Jupyter Notebook UI.
+
+Time Commitment: About 5-10 minutes.
+
+## Prerequisites
+
+Before you start with the conversion, please ensure you have the following:
+
+<details>
+  <summary>Installed Software and Services</summary>
```
Comment (antoniivanov) [inline projects/vdk-plugins/vdk-jupyter/convert-job-tutorial-temp/tutorial.md:10]: The sumamry are not very noticeable. 
make them bold or even better into a heading like that 

```suggestion
  <summary><h3>Developed Data Job</h3></summary>
```

**Review by antoniivanov — APPROVED** [review]

**Comment by duyguHsnHsn** [comment]
Closing this PR and adding the tutorial to Wiki

**Pull request closed without merging**